src/pages/dashboard/Dashboard.tsx

```diff
@@ -126,6 +128,10 @@ const Dashboard: Component<RouteSectionProps> = () => {
     return devices()?.[0]?.dongle_id
   }
 
+  onMount(() => {
+    if (!getAccessToken()) navigate('/login')
```

Comment:
We already have a `<Navigate href="/login">` case in the JSX switch below, it's just *after* we check whether we are visiting a specific route, so it only redirects visits to `/` to `/login`. We can remove that if we have this

I'm not sure there is a need to put this inside `onMount`, we should have the access token immediately